Sources/ArcGISToolkit/Components/FormView/FormViewModel.swift

```diff
@@ -16,53 +16,41 @@ import ArcGIS
 import Combine
 import SwiftUI
 
-/// - Since: 200.3
+/// - Since: 200.4
 public class FormViewModel: ObservableObject {
```

Comment:
Since we're tagging seven separate properties with `@MainActor` it seems like it might make more sense to mark the entire class instead but that led to this issue:

<img width="798" alt="Screenshot 2023-12-12 at 10 12 56" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/491a4a1b-7917-489d-87a1-a5acc3fa8f46">

I experimented and we can get around it by moving the body of `clearIsVisibleTasks()` into `deinit`:
```swift
deinit {
    isVisibleTasks.forEach { task in
        task.cancel()
    }
    isVisibleTasks.removeAll()
}
```

I see `clearIsVisibleTasks()` is called in `initializeIsVisibleTasks()` but if I'm not mistaken there would never be any tasks to clear when that runs anyway.

Reply:
Ok, that works for me. I put the `clearIsVisibleTasks()` there for completeness, but you're correct, it's really not necessary.